### DISCUSSION TITLE
Automatic Lockout expiry

### DISCUSSION THREAD

**Discussion by tyteen4a03** [zitadel/zitadel#8660] [Ideas]
It would be great to have an option where a user's lockout automatically expires after X minutes (with linear / exponential decay), as a protection against brute-force attacks while not requiring operator involvement.

**Comment by fforootd** [comment]
Hm yeah that could make sense as an improved lockout/unlock policy.

I guess it is best to open an issue here https://github.com/zitadel/zitadel/issues/new?assignees=&labels=improvement&projects=&template=improvement.yaml

So that somebody (we or a contributor) can pick it up.
